src/Http/Http.Extensions/gen/StaticRouteHandlerModel/StaticRouteHandlerModel.Emitter.cs

```diff
@@ -44,6 +44,9 @@ public static string EmitVerb(this Endpoint endpoint)
             "MapPost" => "PostVerb",
             "MapDelete" => "DeleteVerb",
             "MapPatch" => "PatchVerb",
+            "MapMethods" => endpoint.Operation.GetMapMethodVerbs(),
```

Comment:
Using `syntax.ToFullString()` for this like `GetMapMethodVerbs()` does seems sketchy. I'm betting this works fine when the `httpMethods` argument doesn't reference anything from an outers cope like in the following case:

```csharp
app.MapMethods("/", new[] { "GET", "POST" }, (HttpContext httpContext, int id) =>
{
```

But what about when the methods are stored in a variable? We should add a test case for this.

```csharp
var myMethods = new[] { "GET", "POST" };
app.MapMethods("/", myMethods, (HttpContext httpContext, int id) =>
{
```

Fortunately since this emitted inside of our generated overload to `MapMethods` we have the `httpMethods` argument already in scope, so we should be able to just flow it through and delete `GetMapMethodVerbs()`.

```suggestion
            "MapMethods" => "httpMethods",
```

Reply:
Oh! This is so much more elegant 🙇🏽